Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -1239,7 +1238,9 @@ class CombinatorialKalmanFilter {
     }
 
     auto propState =
-        m_propagator.template makeState(initialParameters, propOptions);
+        m_propagator.template makeState<start_parameters_t, PropagatorOptions,
+                                        StubPathLimitReached>(initialParameters,
```

Comment:
So this `StubPathLimitReached` disables the propagator path limiter since it replaces the default path limiter that the propagation appends?

Reply:
Yes and the reason I don't set the path limit to infinity is that we actually use it later to handle loop protection for each branch separately